### DISCUSSION TITLE
NCollpaseTransition in render function

### DISCUSSION THREAD

**Discussion by tngnts** [tusen-ai/naive-ui#3739] [Q&A]
Struggling to write a reactive NCollpaseTransition in a render and was wondering if anyone could give me a hand. 

This initially works but any changes to showDetails does not update the NCollapseTransition component and display/hide the details.
`h(NSwitch, {onUpdateValue: () => this.toggleDetails(0)}),
  h(NCollapseTransition, {
    show: this.showDetails,
  },
  () => "Details test copy goes here"),`

**Comment by 07akioni** [comment]
It should be reactive. Maybe you can put a reproduction to let us figure out the problem.
